CONTRIBUTING.md

```diff
@@ -93,24 +99,39 @@ The [Gobwords Style Guide][1] is an index of commonly mis-capitalized, mispunctu
-* Integration
-  * In the context of describing / discussing Agent checks, use “check”.
-* Please
-  * There’s no reason to plead with the reader; maybe they’ll read the docs, maybe they won’t.
-
-## Link
-
-Links must be specified using the reference format (i.e. in the footnote) in order to aid [the translation process][2].
+| Word to avoid        | Workaround                                                                                 |
+| :------------------- | :----------------------------------------------------------------------------------------- |
+| Currently            | just omit it                                                                               |
+| Refer to/visit       | When preceding a link; use “See” or “Read”                                                 |
+| A number of          | This is vague. Slightly less vague: “a few”, “several”, “many”.                            |
+| [in the] Datadog app | No need for the definite article; use "[in] Datadog".                                      |
+| Product              | When referencing Datadog (e.g. “the Datadog product”), omit it or use “service”            |
+| Integration          | In the context of describing / discussing Agent checks, use “check”.                       |
+| Please               | There’s no reason to plead with the reader; maybe they’ll read the docs, maybe they won’t. |
+
+
+## Formating
+
+### Text
+
+Use specific text formating to enhance your content readability.
+
+| Formating   | Rule                                                                                                                    | Example                                               |
+| :---        | :------                                                                                                                 | :----                                                 |
+| `Backquote` | Used every time there is code related content in some text                                                              | Use the `foo` parameter                               |
+| **Bold**    | Subjectively pointing the reader to something that he shouldn't miss.                                                   | **This is important**, not that.                      |
+| *Italic*    | Literal translated words, default values, functionality name, webpage name                                              | Go the the *setting* page in your Datadog application |
+| [Link]      | Links must be specified using the reference format (i.e. in the footnote) in order to aid [the translation process][2]. | `This is a [Link][1]`                                 |
+
+### Fixed-width
+
+By default images are displayed on the full width of a page, if your graphic example doesn't need that much space, use the `style="width:XX%;"` parameter within your Image partial in order to make them better fit your content. 
```

Comment:
Images are displayed on the full width of a page by default. If your image doesn't need to be that large, use the `style="width:XX%;"` parameter within the image partial in order to scale the image proportionally.